src/base/ApplicationVersionedLambda.ts

```diff
@@ -87,16 +87,13 @@ export class ApplicationVersionedLambda extends Resource {
         ].filter((v: string) => v),
       },
       tags: this.config.tags,
+      environment: this.config.environment
```

Comment:
terraform cdk fixed the issue that caused env variables to be lowercased. So we no longer need to use the escape hatch